tests/integration/conftest.py

```diff
@@ -82,12 +82,17 @@ def test_catalog_fixture(make_catalog):
 
 @pytest.fixture
 def make_schema(sql_exec, make_random):
-    def create(*, catalog="hive_metastore"):
-        name = f"{catalog}.ucx_S{make_random(4)}".lower()
-        sql_exec(f"CREATE SCHEMA {name}")
-        return name
+    def create(*, catalog="hive_metastore", schema_name: str | None = None):
```

Comment:
```suggestion
    def create(*, catalog="hive_metastore", schema: str | None = None):
```

for consistency of argument naming. e.g. we have `catalog`, not `catalog_name`.